modules/nf-core/odgi/squeeze/main.nf

```diff
@@ -0,0 +1,39 @@
+process ODGI_SQUEEZE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::odgi=0.8.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/odgi:0.8.2--py39h2add14b_0':
+        'quay.io/biocontainers/odgi:0.8.2--py39h2add14b_0' }"
+
+    input:
+    tuple val(meta), path(graphs)
+
+    output:
+    tuple val(meta), path("*.og"), emit: graph
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    ls *.og >> files
```

Comment:
```suggestion
    ls *.og > files
```
First write should sure to start fresh (not that anything should be getting into the workdir, but still)